purchase.adoc

```diff
@@ -1,675 +1,885 @@
-  "lastPurchaseHash": "1476088362675HKXEKo7EEeaq_0GG8pcFtg",
-  "linkUrls": [
-        "<https://purchase.izettle.com/purchases/v2?limit=10&descending=true&lastPurchaseHash=1476088362675HKXEKo7EEeaq_0GG8pcFtg>; rel=\"next\""
-  ]
-}
-```
-
-#### Example response 3 (discounts)
-```json
-{
-  "purchases": [
-    {
-      "purchaseUUID": "biqV4OiTEea6-larSAFA7w",
-      "amount": 10000,
-      "vatAmount": 1331,
-      "country": "SE",
-      "currency": "SEK",
-      "timestamp": "2017-02-01T15:31:00.648+0000",
-      "purchaseNumber": 1507,
-      "userDisplayName": "Ford Prefect",
-      "userId": 1713921,
-      "organizationId": 14184411,
-      "products": [
-        {
-          "quantity": "2",
-          "productUuid": "39cde734-e893-11e6-b8f1-2d3db21fba6a",
-          "variantUuid": "302b4d5c-e893-11e6-b8f1-2d3db21fba6a",
-          "type": "PRODUCT",
-          "id" : "0",
-          "vatPercentage": 25,
-          "unitPrice": 2000,
-          "rowTaxableAmount": 2240,
-          "name": "Foo",
-          "variantName": "",
+          "productUuid": "c8037b10-5381-11eb-a35d-4df8c9434273",
+          "variantUuid": "f29e9da0-5381-11eb-b308-d53bdad1e1da",
+          "vatPercentage": 12,
+          "unitPrice": 10000,
+          "rowTaxableAmount": 6786,
+          "name": "T-shirt",
+          "description": "Cool T-shirt",
+          "variantName": "Small",
           "discount": {
-            "quantity": 1,
-            "amount": 500
+            "percentage": 20,
+            "quantity": 1
           },
-          "discountValue": 500,
-          "autoGenerated": false,
+          "discountValue": 2000,
+          "comment": "20% off",
+          "type": "PRODUCT",
           "libraryProduct": true
         },
         {
-          "quantity": "10",
-          "productUuid": "4385c5ee-e893-11e6-b8f1-2d3db21fba6a",
-          "variantUuid": "3bdf6232-e893-11e6-b8f1-2d3db21fba6a",
-          "type": "PRODUCT",
-          "id" : "1",
+          "quantity": "1",
+          "productUuid": "c8037b10-5381-11eb-a35d-4df8c9434273",
+          "variantUuid": "f29e9da0-5381-11eb-80a8-3285f2ac85a4",
           "vatPercentage": 12,
-          "unitPrice": 1000,
-          "rowTaxableAmount": 6429,
-          "name": "Bar",
-          "variantName": "",
+          "unitPrice": 10000,
+          "rowTaxableAmount": 6786,
+          "name": "T-shirt",
+          "description": "Cool T-shirt",
+          "variantName": "Large",
           "discount": {
-            "quantity": 1,
-            "percentage": 10
+            "amount": 2000,
+            "quantity": 1
           },
-          "discountValue": 1000,
-          "autoGenerated": false,
+          "discountValue": 2000,
+          "comment": "Fixed amount discount",
+          "type": "PRODUCT",
           "libraryProduct": true
         }
       ],
       "discounts": [
         {
-          "name": "",
-          "percentage": 20,
+          "percentage": 5,
```

Comment:
The cart wide discounts now also has the field "value" which is the value resulting from applying the percentage/amount times the quantity on the item lines.